.github/CONTRIBUTING.md

```diff
@@ -101,30 +101,30 @@ These instructions assume a parent directory `bar` and we will be adding two chi
     $ mkdir bar && cd bar
     # Clone the forked repo locally
     bar $ git clone https://github.com/YOURNAME/botpress.git
+    # Install the dependencies
```

Comment:
after cloning you need to
- `cd botpress`
- `yarn install` to get lerna itself